perf/perf_watch_point.py

```diff
@@ -55,7 +55,8 @@ def setUp(self):
         self.build_dir = os.path.join(self.workdir, 'wptest-master')
         build.make(self.build_dir)
         os.chdir(self.build_dir)
-        process.run("insmod wptest.ko")
+        if not linux_modules.load_module("wptest"):
+            process.run("insmod wptest.ko")
```

Comment:
as the next step, we check if wptest.ko present , if this file not present insmod will fail and script will error out can you please accommodate same